packages/teams-js/src/public/meeting.ts

```diff
@@ -925,6 +926,47 @@ export namespace meeting {
     );
   }
 
+  /**
+   * This function is used to join a meeting
```

Comment:
More detail on what this actually does would be useful to developers. Does it open the meeting in a pop out? Does it navigate away from the application?

Reply:
@TrevorJoelHarris, Added the details of the function.